main/app-menu.js

```diff
@@ -0,0 +1,57 @@
+const { Menu, MenuItem, ipcMain } = require('electron')
+const { ipcMainEvents } = require('./ipc')
+
+function setupAppMenu (/** @type {import('./typings').Context} */ ctx) {
+  // Add "Check for updates..." item to the Application menu on MacOS
+  // GitHub issues tracking the work to add this menu item on other platforms:
+  //  - Windows: https://github.com/filecoin-project/filecoin-station/issues/63
+  //  - Linux: https://github.com/filecoin-project/filecoin-station/issues/64
+  if (process.platform !== 'darwin') return
+
+  const menu = Menu.getApplicationMenu()
+  if (!menu) return
+
+  menu.items[0].submenu?.insert(1, new MenuItem({
+    id: 'checkForUpdates',
+    label: 'Check For Updates...',
+    click: () => { ctx.manualCheckForUpdates() }
+  }))
+  menu.items[0].submenu?.insert(2, new MenuItem({
+    id: 'checkingForUpdates',
+    label: 'Checking For Updates',
+    enabled: false,
+    visible: false
+  }))
+
+  Menu.setApplicationMenu(menu)
+  setupIpcEventListeners(menu)
+}
+
+/**
+ * @param {Electron.Menu} appMenu
+ */
+function setupIpcEventListeners (appMenu) {
+  ipcMain.on(ipcMainEvents.UPDATE_CHECK_STARTED, () => {
+    getItemById('checkForUpdates').visible = false
+    getItemById('checkingForUpdates').visible = true
+  })
+
+  ipcMain.on(ipcMainEvents.UPDATE_CHECK_FINISHED, () => {
+    getItemById('checkForUpdates').visible = true
+    getItemById('checkingForUpdates').visible = false
+  })
+
+  /**
```

Comment:
```suggestion
  /**
   * Get item or fail with a useful error message.
```

Reply:
Done in [af82612](https://github.com/filecoin-project/filecoin-station/pull/62/commits/af8261296e09e04876d64135d36588b2360d2e52)